packager/react-native-xcode.sh

```diff
@@ -10,15 +10,21 @@
 # This script is supposed to be invoked as part of Xcode build process
 # and relies on environment variables (including PWD) set by Xcode
 
+DEST=$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH
+
+if [[ "$CONFIGURATION" = "Debug" && "$PLATFORM_NAME" != "iphonesimulator" ]]; then
+  ipconfig getifaddr en0 > "$DEST/packager-ip.txt"
```

Comment:
Does it make any sense to use a standard plist for this purpose instead of creating a txt file?